app/models/metric/statistic.rb

```diff
@@ -0,0 +1,17 @@
+module Metric::Statistic
+  def self.calculate_stat_columns(obj, attrs, ts = nil)
+    valid_obj = obj.kind_of?(ExtManagementSystem)
+    raise "object #{obj} is not one of ExtManagementSystem" unless valid_obj
```

Comment:
@yaacov this is a duplication of logic. Why maintaining the list of relevant entities in two places?

For as much as I am concerned you could go with:

    return {} unless obj.kind_of?(ExtManagementSystem)

and remove the check above (in `rollup.rb`).